web/src/settings_org.ts

```diff
@@ -562,29 +612,39 @@ export function discard_realm_property_element_changes(elem) {
             break;
         default:
             if (property_value !== undefined) {
-                settings_components.set_input_element_value($elem, property_value);
+                // TODO(evy) manually test all the settings before merging, just in case
+                // there are other types that can end up here.
```

Comment:
Yeah, there should be not be any other type for this block, but might be worth to test manually.

Reply:
Tested manually and it seemed fine!